### ISSUE TITLE
Search Icon in the Bottom Toolbar

### ISSUE THREAD

**Issue by alphanerdgen** [msoultanidis/quillnote#80]
Would be easier to access from there instead of the 3 dots menu (which is harder to reach on big phones).

**Comment by Pranab009** [comment]
Yeah. Please sir  move down hamburger menu & three dot menu & search bar.
Merged task tab into plus"+" button
When press plus button then show note and task.

![Screenshot_20210731-072308_1](https://user-images.githubusercontent.com/65340361/127798112-b9ab85fd-c730-499d-ba7d-1f6028a334ee.png)

![Screenshot_20210802-081620](https://user-images.githubusercontent.com/65340361/127798134-c7798f87-7f12-423a-a6ef-417f2ebe9ba7.png)

**Comment by Kodmod** [comment]
We desperately need this!
